src/components/Book.js

```diff
@@ -1,3 +1,25 @@
-function Book() {}
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
+function Book({ id, title, author }) {
+  const dispatch = useDispatch();
+  return (
+    <div className="book-item">
+      <p className="title">{title}:</p>
+      <p className="author">{author}</p>
+      <button
+        type="button"
+        onClick={() => {
+          dispatch(removeBook(id));
```

Comment:
- [ ] @PNdunguMaina instead of adding an action creator directly in your return, kindly create a method in your component to get the id that dispatched the action creator.